Makefile

```diff
@@ -7,8 +7,8 @@ install-tools:
 
 VERIFY += generate
 .PHONY: generate
-generate: CODE_GEN_DIR = pkg/json
-generate: GEN = pkg/cmd/codegenerator/generator.go
+generate: CODE_GEN_DIR = pkg/json/schemas
```

Comment:
the generated types are not schemas, they are possible types used by the system

Reply:
moved to pkg/types